src/pybind/mgr/volumes/module.py

```diff
@@ -561,6 +561,12 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Cancel an pending or ongoing clone operation.",
             'perm': 'r'
         },
+        {
+            'cmd': 'fs purge status '
+                   'name=vol_name,type=CephString ',
+            'desc': "Get status of a purging of volume trashcan",
+            'perm': 'r'
+        },
```

Comment:
This looks like it's doing the right things. I would like to see example command output. That could go in the docs in an "Introspection" section.

I'm also wondering where the progress bar in `ceph status` is wired up? (See ticket!)